assignments/array-methods.js

```diff
@@ -64,25 +64,41 @@ runners.forEach(function (runner) {
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = runners.map(function(runner){
-    return runners.first_name
+    return runner.first_name.toUpperCase();
 })
-console.log(allCaps); 
+//console.log(allCaps); 
 
 // ==== Challenge 3: Use .filter() ====
 // The large shirts won't be available for the event due to an ordering issue.  Get a list of runners with large sized shirts so they can choose a different size. Return an array named largeShirts that contains information about the runners that have a shirt size of L and log the result
-let largeShirts = [];
-console.log(largeShirts);
+let largeShirts = runners.filter(function (runner) {
```

Comment:
Nicely done. Although this is a preference because ES5 function is good either way, but try transitioning into ES6 arrow syntax and using it more often.